### DISCUSSION TITLE
No data back just after prompt input- Storybook

### DISCUSSION THREAD

**Discussion by JayAIsea** [AntlerVC/firetable#1334] [Support / Q&A] [answered]
I've been so excited with Storybook template, but found the same issues from some people with the above.
I created a table and changed the prompt but I don't get any data for other columns, what could have gone wrong?
Below youtube link is just for your easy reference.

https://youtu.be/2MhD4dRbl3g?t=797

**Comment by BeeBombshell** [comment, marked as answer]
Hey, @JayAIsea! Would love to help you out with this. Just to clarify, are you not seeing a response object generated by OpenAI when you enter your prompt? One possible reason could be that you haven't configured your secrets with the correct Secret Name. To address this, I recommend following the step-by-step guide provided in this blog post: https://www.rowy.io/blog/storybooks-bravo.

If this still doesn't work, could you share a recording via https://www.jam.dev so that we can take a look?
